modules/katamari-assertions/src/main/ts/ephox/katamari-assertions/api/KAssert.ts

```
@@ -2,10 +2,11 @@ import { Assert, type TestLabel } from '@ephox/bedrock-client';
 import { Testable } from '@ephox/dispute';
 import { Optional, OptionalInstances, Result, ResultInstances } from '@ephox/katamari';
 
+type Testable<A> = Testable.Testable<A>;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Testable'. Consider to delete it or rename it.

'Testable' is defined here:


(lint/suspicious/noRedeclare)